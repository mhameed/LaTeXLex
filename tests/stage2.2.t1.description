author: ehc
date: 09/06/22
description: checks that the tabs that InftyReader creates are correctly dealt with by the tabbing stage NB these are not tabs such as those found in the human written tabbing environment and are not confined to the tabbing environment