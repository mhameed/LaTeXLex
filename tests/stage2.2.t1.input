[Tab]
[Tab][Tab]
[Tab][Tab][Tab]

 [Tab] [Tab] [Tab]

Here is some text and then a [Tab] and then two with no space between [Tab][Tab] and then two with spaces between [Tab] [Tab].

[Tab]A
[Tab][Tab]B
[Tab][Tab][Tab]C
