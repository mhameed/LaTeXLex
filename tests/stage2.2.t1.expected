(1 tab) 
(2 tab) 
(3 tab) 

 (1 tab) (1 tab) (1 tab) 

Here is some text and then a (1 tab) and then two with no space between (2 tab) and then two with spaces between (1 tab) (1 tab) .

(1 tab) A
(2 tab) B
(3 tab) C
